### PR TITLE
Kernel/SVC: Implemented svcQueryMemory.

### PR DESCRIPTION
**Pull request by Subv**

### CODE REVIEW

File: src/core/hle/svc.cpp [hunk 1]
```diff
@@ -530,8 +531,19 @@ static ResultCode ReleaseSemaphore(s32* count, Handle handle, s32 release_count)
 }
 
 /// Query memory
-static ResultCode QueryMemory(void* info, void* out, u32 addr) {
-    LOG_ERROR(Kernel_SVC, "(UNIMPLEMENTED) called addr=0x%08X", addr);
+static ResultCode QueryMemory(MemoryInfo* memory_info, PageInfo* page_info, u32 addr) {
+    auto vma = Kernel::g_current_process->address_space->FindVMA(addr);
+
+    if (vma == Kernel::g_current_process->address_space->vma_map.end())
+        return ResultCode(-1); // TODO(Subv): Find right error code
```
Comment (yuriks) [inline src/core/hle/svc.cpp]: You have a demo already, should take almost no time to find this one. :P

**Comment by yuriks** [comment]
Btw, this was tested with: https://github.com/yuriks/3ds-demos

Demo Run

**Comment by yuriks** [comment]
Other than the error, LGTM

**Comment by mailwl** [comment]
looks like games doesn't check actual result, so i think, the error result may be somethink like "1013 	Invalid address", or "1021 	Out of range "

**Comment by bunnei** [comment]
LGTM

**Comment by Subv** [comment]
Updated with the correct error code

**Pull request merged**